### ISSUE TITLE
scope yaml vignette

### ISSUE THREAD

**Issue by jonthegeek** [yonicd/slackteams#15]
As a slackteams user, I'd like help setting up a yaml for custom scopes, so that I can semi-easily apply slackteams to an unexpected usecase.

**Comment by jonthegeek** [comment]
Include basics on including such a yaml in a derived package.

**Comment by bkmgit** [comment]
It would be helpful to extend the vignette a little. At present it is not immediately obvious how to use the slackverse scope.

**Comment by yonicd** [comment]
Thanks for the feedback.

Can you expand on the type of content you think is missing that would help.

**Comment by bkmgit** [comment]
Can find the file scopes.yml in my installation. However my first attempts at getting expanded scopes seem not to work. Would like to do something like:
* slackteams::add_team_interactive( scopes=load_scopes("/Path/To/scopes.yml","base"))
* slackteams::add_team_interactive( scopes=load_scopes("users:read", "channels:read", "groups:read", "im:read", "mpim:read"))

But it seems neither of these work.

**Comment by bkmgit** [comment]
The following work,
slackteams::add_team_interactive(scopes=load_scopes(system.file("scopes.yml", package = "slackteams"),"base"))
slackteams::add_team_interactive(scopes=load_scopes(system.file("scopes.yml", package = "slackteams"),"slackverse"))
 Will make a pull request with suggested improvements
Thanks for the useful package

**Comment by yonicd** [comment]
The default for `load_scopes` is the system file location 

https://github.com/yonicd/slackteams/blob/master/R/scope_managment.R#L22_L26

so you would call it like so

```r
load_scopes(which = 'base')
load_scopes(which = 'slackverse')
```

It may be easier to switch the order of the arguments in the function where the `which` comes first, then you would call it like this

```r
load_scopes('base')
load_scopes('slackverse')
```

In your case that you tried:
```r
slackteams::add_team_interactive( scopes=load_scopes("users:read", "channels:read", "groups:read", "im:read", "mpim:read"))
```
you are trying to pass in scopes where the path to a file should be. Currently, there is no mechanism to accommodate that since the premise of the scopes was to have a persistent yml file(s). But that seems like a reasonable addition.

**Comment by bkmgit** [comment]
The file is ok, just was not so clear to me that it is the main choice. Modifying pull request.

**Comment by jonthegeek** [comment]
Resolved in #30
